### PR TITLE
Fix naming bag AccountsController

### LINKED ISSUE #783: Naming error in AccountsController
Change property name 'FisrtName' to 'FirstName' in AccountsController (Api) 
image
Synchronize changes with the frontend.

### PR DESCRIPTION
**Pull request by Ssashaa**
I have fixed the naming error in the AccountsController and updated the example in SwaggerExamples (SingInResponse)

### CODE REVIEW

File: CharlieBackend.Api/SwaggerExamples/AccountsController/SignInResponse.cs [hunk 1]
```diff
@@ -20,10 +21,11 @@ public SignInResponse GetExamples()
                 first_name = "Evgeniy",
                 last_name = "Karnaushekno",
                 role = 1,
-                id = 15
+                role_list = new Dictionary<string, string>()
+                {
+                    {"student", "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJodHRwOi8vc2NoZW1hcy5taWNyb3NvZnQuY29tL3dzLzIwMDgvMDYvaWRlbnRpdHkvY2xhaW1zL3JvbGUiOiJTdHVkZW50IiwiSWQiOiI0IiwiRW1haWwiOiJ0aG9tYXMucm9iZXJ0c0BleGFtcGxlLmNvbSIsIkFjY291bnRJZCI6IjkiLCJuYmYiOjE2NDM3MTIyNDEsImV4cCI6MTY0Mzc1NTQ0MSwiaXNzIjoiTXlBdXRoU2VydmVyIiwiYXVkIjoiTXlBdXRoQ2xpZW50In0.1ITyYhlGsykxSJj8ftPLmYTV2Fik04dej6QX7HcEmyc" }
```
Comment (ArsenMaistryshyn) [inline CharlieBackend.Api/SwaggerExamples/AccountsController/SignInResponse.cs]: Is this token fake?

Comment (Ssashaa) [reply CharlieBackend.Api/SwaggerExamples/AccountsController/SignInResponse.cs]: No, this is a real example token

Comment (ArsenMaistryshyn) [reply CharlieBackend.Api/SwaggerExamples/AccountsController/SignInResponse.cs]: please use fake data for examples

**Review by BobTheSuperman — APPROVED** [review]

**Review by DmitryTkachenko — APPROVED** [review]

File: CharlieBackend.Api/SwaggerExamples/AccountsController/SignInResponse.cs (hunk 1 above)
Comment (Ssashaa) [reply CharlieBackend.Api/SwaggerExamples/AccountsController/SignInResponse.cs]: Okey, thanks for the comment

**Review by ArsenMaistryshyn — APPROVED** [review]

**Pull request merged**